### ISSUE TITLE
CD Audio is stuttering and making the game jittery in 64-bit build

### ISSUE THREAD

**Issue by augmentedfourth** [alunbestor/Boxer#107]
I'm playing Rayman using a Boxer image created by 32-bit Boxer. The game uses CD Audio tracks to play the soundtrack, but Boxer did the awesome right thing of ripping the CD audio into the image so the game music will work. (I used an actual CD copy of the game for Boxer to create the image.)

However, the game is stuttering and really jittery to the point where it's unplayable. Maybe this is related to the CD audio access?

**Comment by augmentedfourth** [comment]
Note that I'm using the "maddsV2 12.07.2019.zip" 64-bit build.

**Comment by almeath** [comment]
Do you get the same result regardless of the number of CPU cycles you set? What about dynamic vs normal core? (in Boxer that is toggled by the “optimize for newer games” setting)

**Comment by augmentedfourth** [comment]
The "optimize for newer games" helped a lot, and I was able to fiddle with the CPU cycles to get it feeling mostly normal. I also had to play with the CPU cycles to get the speed to a fairly comfortable place. It's not perfect, and sometimes it feels like the game is speeding up & slowing down a bit, but it's mostly playable.

I also had to disable the joystick entirely in Boxer. I originally had just set Enjoyable to just handle the axes, and allow the joystick button input to pass through to the game. But there must have been some sort of "ghost input" from the axes that was jerking around a lot.

It's pretty frustrating to have to tweak all this stuff just to get to "mostly playable" when 32-bit Boxer works flawlessly right out of the box. The main reason I use Boxer rather than DOSBox directly is specifically to avoid all the esoteric configuration when I feel like playing my older games.

**Comment by almeath** [comment]
Yes, CPU cycles and core settings are the most common cause of sound stuttering, particularly on faster machines. Sometimes nothing works perfectly, and it takes a lot of tweaking, that is just the nature of DOS emulation. Generally, for sound issues you should try:

1. Cap the CPU cycles or go into the config and set it to “auto”.
2. Try the normal core, or go into the config and set it to “auto”.
3. If using sound blaster, try different types.
4. Try different graphic modes, being the maximum that the game supports, i.e EGA, VGAONLY, SVGA_S3 (this has fixed PC speaker sound in some games for me)
5. Alter the sound prebuffer setting; I generally set it to 80 and that has avoided stuttering in some games.
